modules/apps/transfer/keeper/msg_server.go

```diff
@@ -5,18 +5,32 @@ import (
 
 	errorsmod "cosmossdk.io/errors"
 	sdk "github.com/cosmos/cosmos-sdk/types"
+	govtypes "github.com/cosmos/cosmos-sdk/x/gov/types"
 
 	ibcerrors "github.com/cosmos/ibc-go/v7/internal/errors"
 	"github.com/cosmos/ibc-go/v7/modules/apps/transfer/types"
 )
 
 var _ types.MsgServer = (*Keeper)(nil)
 
+func (k Keeper) UpdateParams(goCtx context.Context, msg *types.MsgUpdateParams) (*types.MsgUpdateParamsResponse, error) {
```

Comment:
could we move `UpdateParams` to below `Transfer` at the bottom of the file

Reply:
implemented